client/src/components/ServicesFilter/ServicesFilter.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {listItemSectionBuilder, MultiSelectCheckbox, TableToolbar} from 'wix-style-react';
+import {translate} from '../../utils';
+import {isEqual} from 'lodash';
+
+export default class ServicesFilter extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            selectedServices: props.filters.services
+        };
+    }
+
+    _getOptionsList = (options) => {
+        const {servicesGroups} = this.props;
+        for (const serviceGroup in servicesGroups) {
+            if (servicesGroups[serviceGroup].length) {
+                options.push(listItemSectionBuilder({title: translate(`ServicesFilter.${serviceGroup.toLowerCase()}`)}));
+                options.push(...servicesGroups[serviceGroup]);
+            }
+        }
+    };
+
+    _onServiceOptionSelect = (optionScheduleId) => {
+        this.setState({selectedServices: [...this.state.selectedServices || [], optionScheduleId]});
+    };
+
+    _onServiceOptionDeselect = (optionScheduleId) => {
+        this.setState({selectedServices: this.state.selectedServices.filter((selectedService) => selectedService !== optionScheduleId)});
+    };
+
+    _onServiceFilterClose = () => {
+        const {onFilterChanged, filters} = this.props;
+        if (!isEqual(filters.services, this.state.selectedServices)) {
+            onFilterChanged('services', this.state.selectedServices);
+        }
+    };
+
+    render() {
+        const options = [];
+        this._getOptionsList(options);
+        return (
+            <TableToolbar.Item>
```

Comment:
the \<TableToolbar.Item> and \<TableToolbar.Label> does not belong here.
should go on the BookingsList component.

think if you wanted to use this ServicesFilter elsewhere that isn't need this wrapping.